cameraMatrix:
          [805.59377,   0.     , 662.62367,
           0.     , 805.56467, 424.54258,
           0.     ,   0.     ,   1.     ]
cameraDistCoeffs: [-0.314770, 0.081361, 0.000911, 0.000117, 0.000000]